docs/production/deploying-to-cloud/aws/aws-ecs.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "AWS Elastic Container Service"
```

Comment:
Can you name the doc "AWS ECS deployment architecture"? The sidebarTitle can be "Architecture"